packages/loaders/src/Progress.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { keyframes } from 'styled-components';
+
+const COMPONENT_ID = 'loaders.progress';
+
+const { zdColorBlue600, zdColorGrey200, zdSpacingXs } = require('@zendeskgarden/css-variables');
+
+const heights = {
+  small: 2,
+  medium: 6,
+  large: 12
+};
+
+const sizeToBorderRadius = size => heights[size] / 2;
+
+const ProgressBarContainer = styled.div.attrs(props => ({
```

Comment:
What does everyone else think about the "container" in the name of this component? One concern might be that consumers of Garden may get this component confused with similarly named `react-containers` components, which are light weight render prop components that wrap a custom React hook. 

```js
useBreadcrumb() -> <BreadcrumbContainer>
useSelection() -> <SelectionContainer>
useAccordion() -> <AccordionContainer>
useTabs() -> <TabsContainer>

??? -> <ProgressBarContainer>
```

Or does "container" mean wrapping something (anything), and in this case, `<ProgressBarContainer>` wraps `<ProgressBarIndicator>` with CSS?

Reply:
Yes, you are right that might be a bit confusing. It is just the wrapper element. What about `ProgressBackground`?

Reply:
I changed the naming to `ProgressBackground` and `ProgressIndicator`.

Reply:
Awesome @sunesimonsen, that sounds great!